src/main/kotlin/commands/Nick.kt

```diff
@@ -89,10 +89,17 @@ class Nick(private val chattORE: ChattORE) : BaseCommand() {
 
     @Subcommand("presets")
     @CommandPermission("chattore.nick.preset")
-    @CommandCompletion("@username")
-    fun presets(player: Player, @Optional shownText: String?) {
+    @CommandCompletion("pride|birdflop|players")
+    fun presets(player: Player, @Optional shownText: String?, @Optional type: String?) {
+        val presets = when (type?.lowercase()) {
+            "pride" -> pridePresets
+            "birdflop" -> birdflopPresets
+            "players" -> playerPresets
+            else -> mapOf()
+        }
+        
         val renderedPresets = ArrayList<Component>()
-        for ((presetName, preset) in chattORE.config[ChattORESpec.nicknamePresets]) {
+        for ((presetName, preset) in presets) {
```

Comment:
Because of this modification, the config file will never be read when loading. This won't allow us to modify _just_ the config to change what presets are available.

Reply:
Yea didnt think abt that